sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/OperatorOverloadsShouldHaveNamedAlternatives.cs

```diff
@@ -0,0 +1,89 @@
+namespace Tests.Diagnostics

```

Comment:
The rule does not enforce that every operator has a method. For example, if class has unary and binary operator+, having only one Add method would satisfy the rule. As Amaury mentioned, we are also not checking parameters.

I think these limitations are fair, but let's add tests that document that (ideally with short comments).